Service worker manifest test fails on windows

The error is as follows:

```
  1) Basic end-to-end Workflow Produces a service worker manifest after initial build:
     AssertionError: expected [ Array(51) ] to include '/thirdparty/libs.js'
```

It seems to be because it is not using the system separator. Example from generated manifest:

```
(...)
# sw.file.hash: cb85b0bd48234ba4f3b5fe9b4880d2917ea885e1
/typings.d.ts
# sw.file.hash: f72705562b9b90f9c1cc7d24d1558f4310824ed8
/vendor\angular2\bundles\angular2-polyfills.js
# sw.file.hash: 1ff5ae25b83b613a9f2e18bceb6339dfdf547a17
/vendor\angular2\bundles\angular2.dev.js
(...)
```

The test:

```
expect(lines).to.include('/thirdparty/libs.js');
```

--- a/lib/broccoli/service-worker-manifest.js
+++ b/lib/broccoli/service-worker-manifest.js
@@ -16,7 +16,7 @@ class DiffingSWManifest {
     this.options = options;
     this.firstBuild = true;
   }
-  
+
   rebuild(diff) {
     var manifest = {};
     if (this.firstBuild) {
@@ -25,10 +25,10 @@ class DiffingSWManifest {
       // Read manifest from disk.
       manifest = this.readManifestFromCache();
     }
-    
+
     // Remove manifest entries for files that are no longer present.
     diff.removedPaths.forEach((file) => delete manifest[file]);
-    
+
     // Merge the lists of added and changed paths and update their hashes in the manifest.
     []
       .concat(diff.addedPaths)
@@ -60,7 +60,7 @@ class DiffingSWManifest {
     var files = Object.keys(manifest).sort();
     var bundleHash = this.computeBundleHash(files, manifest);
     var contents = files
-      .map((file) => `# sw.file.hash: ${this.computeFileHash(file)}\n/${file}`)
+      .map((file) => `# sw.file.hash: ${this.computeFileHash(file)}\n${path.sep}${file}`)
       .join('\n');
     return `CACHE MANIFEST
 # sw.bundle: ng-cli

--- a/tests/e2e/e2e_workflow.spec.js
+++ b/tests/e2e/e2e_workflow.spec.js
@@ -76,10 +76,10 @@ describe('Basic end-to-end Workflow', function () {
       .readFileSync(manifestPath, {encoding: 'utf8'})
       .trim()
       .split('\n');
-      
+
     // Check that a few critical files have been detected.
-    expect(lines).to.include('/index.html');
-    expect(lines).to.include('/thirdparty/libs.js');
+    expect(lines).to.include(`${path.sep}index.html`);
+    expect(lines).to.include(`${path.sep}thirdparty${path.sep}libs.js`);
   });
 
   it('Perform `ng test` after initial build', function() {
@@ -193,20 +193,20 @@ describe('Basic end-to-end Workflow', function () {
       // sh.rm('-rf', './tmp'); // tmp.teardown takes too long
     });
   });
-  
+
   it('Turn on `noImplicitAny` in tsconfig.json and rebuild', function (done) {
     this.timeout(420000);
-    
+
     var configFilePath = path.join(process.cwd(), 'src', 'tsconfig.json');
     fs.readFile(configFilePath, 'utf8', function(err, data){
-      
+
       var config = JSON.parse(data);
       config.compilerOptions.noImplicitAny = true;
-      
+
       fs.writeFile(configFilePath, JSON.stringify(config), function(){
         //clear the dist folder
         sh.rm('-rf', path.join(process.cwd(), 'dist'));
-        
+
         return ng([
             'build',
             '--silent'

